README.md

```diff
@@ -3,11 +3,12 @@
 ElasticBox CI Plug-in
 =====================
 
-The ElasticBox CI Jenkins plug-in provides full integration between Jenkins and ElasticBox (http://elasticbox.com).
-With this plugin, Jenkins can launch, provision, and manage Jenkins slaves on-demand in different cloud providers via ElasticBox. It also provides build steps to deploy and manage your applications, including complex, multi-tiear applications that are defined as boxes in ElasticBox.
+The ElasticBox CI Jenkins plug-in provides full integration between Jenkins and ElasticBox (http://elasticbox.com).  
```

Comment:
Please, at least also include "Cloud Application Manager" in this description, and add an action item to rename ElasticBox CI to CAM to your list later on.